src/sqlfluff/dialects/dialect_sparksql.py

```diff
@@ -1631,27 +1631,6 @@ class LateralViewClauseSegment(BaseSegment):
     )
 
 
-class OverClauseSegment(ansi.OverClauseSegment):
```

Comment:
Why was this removed?

Reply:
Because the reason it was overridden was to add the thing we’re now adding in ANSI. So no need for override anymore.